options/locale/locale_en-US.ini

```diff
@@ -843,7 +843,7 @@ oauth2_client_id = Client ID
 oauth2_client_secret = Client Secret
 oauth2_regenerate_secret = Regenerate Secret
 oauth2_regenerate_secret_hint = Lost your secret?
-oauth2_client_secret_hint = The secret won't be visible if you revisit this page. Please save your secret.
+oauth2_client_secret_hint = The secret will disappear as soon as you leave or refresh this page. Please save your secret.
```

Comment:
"disappear" reads like "the secret will be unavailable / will not exist" (IIRC "disappear" means "not exist")

Maybe it could be: "The secret is shown only once on this page" ?

Reply:
I also thought of that, but thought that `Please save your secret.` essentially fixes this ambiguity.

Reply:
Marking this as resolved.